verticapy/performance/vertica/collection/collection_tables.py

```diff
@@ -1222,6 +1331,25 @@ def get_pandas_column_type_adjustments(self) -> Mapping[str, str]:
             "sent_bytes": "Int64",
         }
 
+    def get_export_sql(self):
+        return f"""SELECT
```

Comment:
Make this string more readable please

Reply:
I ran out of time to change this formatting.